src/Entities/Addresses.php

```diff
@@ -0,0 +1,204 @@
+<?php
+/**
```

Comment:
remember to only include the entities that are used in this PR (it seems like there are more than the last PR)

Reply:
This is used in the PharmaciesRepository line 14